src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromArtsy.tsx

```diff
@@ -4,6 +4,12 @@ import { createFragmentContainer, graphql } from "react-relay"
 import { ReadMore } from "Styleguide/Components"
 
 import { ArtworkDetailsAboutTheWorkFromArtsy_artwork } from "__generated__/ArtworkDetailsAboutTheWorkFromArtsy_artwork.graphql"
+import { Responsive } from "Utils/Responsive"
+
+export const READ_MORE_MAX_CHARS = {
+  xs: 100,
+  rest: 320,
```

Comment:
can 320 be named `default`. I think that is the more recent convention we are trying to follow with naming. Or maybe even set the bigger one as `small` and set default pointing to small.  I think there was a discussion about that in #design-system slack (https://artsy.slack.com/archives/C9XJKPY9W/p1538150439000100)

Reply:
I think `default` is a little clearer than `small` for those looking in -- updated!